src/core/prod/locals.tf

```diff
@@ -1,21 +1,18 @@
 locals {
   prefix                   = "io"
   env_short                = "p"
-  location_short           = "itn"
+  location_short           = { westeurope = "weu", italynorth = "itn" }
   secondary_location_short = "gwc"
```

Comment:
I'd do the same as you already done with `location_short`, including `gwc` and `neu`